src/main/java/de/kekru/struktogrammeditor/control/GlobalSettings.java

```diff
@@ -108,53 +112,39 @@ private static void readBuildInfoFile(){
 				SimpleDateFormat sdf = new SimpleDateFormat("dd.MM.yyyy HH:mm:ss");
 				buildInfoBuildTime = sdf.format(new Date(Long.parseLong(s)));
 			}
-
 		} catch (RuntimeException e){
 			e.printStackTrace();
 		}
 	}
 	
 	
-	public static String[] getCurrentElementBeschriftungsstil(){
+	public static String[] getCurrentElementBeschriftungsstil() {
 		return elementAuswahlBeschriftungen[beschriftungsStilAktuell >= 0 && beschriftungsStilAktuell < elementAuswahlBeschriftungen.length ? beschriftungsStilAktuell : 0];
 	}
 	
 	
-	private static void loadSettings(){
+	private static void loadSettings () {
 		//Wenn eine alte Einstellungsdatei (bis einschließlich Version 1.4) existiert, diese laden...
-		File f = new File(einstellungsDateiPfadBisVersion1Punkt4);
-		if(f.exists()){
-			String[] einstellungsdaten = Helpers.readTextFile(einstellungsDateiPfadBisVersion1Punkt4);
+		if(einstellungsDateiBisVersion1Punkt4.exists()){
+			List<String> einstellungen = Helpers.readTextFile(einstellungsDateiBisVersion1Punkt4);
 
-			if (einstellungsdaten != null){
-				for(int i=0; i < einstellungsdaten.length; i++){
-					elementBeschriftungenZumEinfuegenInDasStruktogramm[i] = einstellungsdaten[i];
+			if (einstellungen != null){
+				for(String einstellung : einstellungen){
+					elementBeschriftungenZumEinfuegenInDasStruktogramm.add(einstellung);

```

Comment:
Wouldn't it be easier to just write `elementBeschriftungenZumEinfuegenInDasStruktogramm = einstellungen` ?

Reply:
We could also scale it down to just 

`elementBeschriftungenZumEinfuegenInDasStruktogramm = Helpers.readTextFile(einstellungsDateiBisVersion1Punkt4)`